index.bs

```diff
@@ -4612,9 +4640,9 @@ It takes the following input parameters:
 : |requireResidentKey|
 :: The [=effective resident key requirement for credential creation=], a Boolean value determined by the [=client=].
 : |requireUserPresence|
-:: The constant Boolean value [TRUE].
+:: The constant Boolean value [TRUE] unless <code>|options|.{{CredentialCreationOptions/mediation}}</code> is set to {{CredentialMediationRequirement/conditional}}.
```

Comment:
Would there be a case of `requireUserPresence` still getting set to `True` even if conditional mediation is present in the `.create()` options? If not perhaps we more narrowly define when this line to require `False` here:
```suggestion
:: The constant Boolean value [TRUE], or [FALSE] when <code>|options|.{{CredentialCreationOptions/mediation}}</code> is set to {{CredentialMediationRequirement/conditional}} and the user agent previously collected consent from the user.
```